pallets/committee/src/lib.rs

```diff
@@ -115,6 +126,29 @@ pub mod pallet {
         OptionQuery,
     >;
 
+    #[pallet::genesis_config]
+    pub struct GenesisConfig<T: Config> {
+        pub members: Vec<T::AccountId>,
```

Comment:
```suggestion
        pub council_members: Vec<T::AccountId>,
```
Maybe we should also make it possible to set `constituent` members in genesis?